src/createServer.js

```diff
@@ -1,13 +1,29 @@
 'use strict';
 
-// const express = require('express');
+const userServices = require('../src/sevices/users.service');
+const userRouter = require('../src/routes/user.routes');
+const expensesRouter = require('../src/routes/expenses.routes');
+const expensesServises = require('../src/sevices/expenses.service');
+
+const express = require('express');
+const PORT = 5000;
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  userServices.clear();
+  expensesServises.clear();
+
+  app.use('/users', userRouter);
+  app.use('/expenses', expensesRouter);
```

Comment:
you can add middleware express.json() here, so you wont need them in your routes
```suggestion
  app.use('/users', express.json(), userRouter);
  app.use('/expenses', express.json(), expensesRouter);
```

Reply:
Fix this